backend/app/models/guardrail_config.py

```
@@ -20,12 +21,14 @@
     validators: List[ValidatorConfigItem]
 
-class GuardrailOutputRequest(SQLModel):
-    request_id: str
-    output: str
-    validators: List[ValidatorConfigItem]
\ No newline at end of file
+#---------------Guardrail Response Model-----------------#
+class GuardrailResponse(SQLModel):
+    response_id: UUID
+    rephrase_needed: bool = False
+    safe_text: Optional[str] = None
```

Ruff (0.14.14)

No newline at end of file

Add trailing newline

(W292)